partcad/src/partcad/part_factory_feature_ai.py

```
@@ -683,15 +687,13 @@
-                ):
+                coro = part.get_wrapped(self.ctx)
+                with pc_logging.Action("Instantiate", part.project_name, part.name):
                     shape = asyncio.run(coro)
-                if not shape is None:
+                if shape is not None:
                     try:
                         # Best effort to provide an interactive experience
-                        part.show()
+                        part.show(self.ctx)
                     except Exception as e:
                         pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)